### DISCUSSION TITLE
Add possibility to change the account order

### DISCUSSION THREAD

**Discussion by brahimmouhamou** [zoidsh/meru#321] [Feature Requests]
Current version: 3.0.0-alpha.34

### Feature request:
When adding multiple accounts, it seems that you cant change the order of the tabs.
It would be awesome that a user could rearrange the order so that the most import email account is shown in the first tab.

PS: I love this tool! ❤️

**Comment by timche** [comment]
This is actually on my to do list, but I can't give any date when it will come unfortunately. Stay tuned though!
